config/database.yml

```diff
@@ -1,6 +1,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: <%= ENV.fetch("POSTGRES_HOST", "localhost")%>
+  username: <%= ENV.fetch("POSTGRES_USER", "postgres")%>
```

Comment:
```yaml
host: <%= ENV.fetch("POSTGRES_HOST") { "" } %>
username: <%= ENV.fetch("POSTGRES_USERNAME") { "" } %>
```

By Rails default, the`host` and `username` was empty, we need keep the Rails default behaviors.

Reply:
ok